.ruby-version

```diff
@@ -1 +1 @@
-2.7.8
+3.2.2
```

Comment:
Post-install message from dnsruby:
Installing dnsruby...
  For issues and source code: https://github.com/alexdalitz/dnsruby
  For general discussion (please tell us how you use dnsruby): https://groups.google.com/forum/#!forum/dnsruby